### PR TITLE
deb: merge ceph-fs-common into ceph-common

### PR DESCRIPTION
**Pull request by smithfarm**
To be merged together with https://github.com/ceph/ceph-qa-suite/pull/1098

Fixes: http://tracker.ceph.com/issues/16808
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by jcsp** [comment]
:+1:

**Comment by smithfarm** [comment]
Pushed `wip-16808` to gitbuilders.

File: debian/control [hunk 1]
```diff
@@ -116,8 +116,7 @@ Description: common ceph daemon libraries and management tools
 Package: ceph-mds
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}), ${misc:Depends}, ${shlibs:Depends}
-Recommends: ceph-fs-common (= ${binary:Version}),
-            ceph-fuse (= ${binary:Version}),
+Recommends: ceph-fuse (= ${binary:Version}),
```
Comment (tchaikov) [inline debian/control]: this is the 11st scenario in Renaming_a_Package[1]. so probably, we should mark ceph-common 

> Breaks: ceph-fs-common (<< 11.0) (for backport ease)
> Replaces: ceph-fs-common (<< 11.0)

i don't think we need the "Provides: ceph-fs-common"  flag, as the only package depends on ceph-fs-common is ceph-fs-common-dbg. and apparently, they version should be identical. so this does not apply in this case.

---
[1] https://wiki.debian.org/PackageTransition

**Comment by smithfarm** [comment]
@tchaikov Added breaks and replaces. How does it look to you, now?

**Comment by tchaikov** [comment]
@smithfarm probably we can also remove `ceph-fs-common*` from `qa/qa-scripts/cephscrub.sh`?

@tmuthamizhan is `cephscrub.sh` still being used? i grep'ed it in ceph-qa-suite, teuthology, and ceph, no luck.

**Comment by tchaikov** [comment]
lgtm

**Comment by smithfarm** [comment]
Testing with <code>cw -v ceph-qa-suite --simultaneous-jobs 20 --suite rados --filter="singleton-nomsgr/all/16113.yaml" --ceph-qa-suite-git-url http://github.com/SUSE/ceph-qa-suite --suite-branch wip-16808 --ceph-git-url http://github.com/SUSE/ceph --ceph wip-16808 --teuthology-git-url http://github.com/SUSE/teuthology --teuthology-branch wip-opensuse</code>

**running** http://158.69.66.4:8081/ubuntu-2016-07-26_17:58:02-rados-wip-16808---basic-openstack/

**Comment by tchaikov** [comment]
test passed, merging.

**Comment by smithfarm** [comment]
`teuthology-suite --suite upgrade/jewel-x --filter="parallel" --machine-type vps --priority 99 --ceph wip-16808 --suite-branch wip-16808 distros/all/ubuntu_14.04.yaml`

**running** http://pulpito.ceph.com/smithfarm-2016-07-26_22:55:10-upgrade:jewel-x-wip-16808---basic-vps/

**Pull request merged**